### PR TITLE
Clean unused code for already deleted {N} versions

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
Since NativeScript version 1.5.2 is obsolete and it cannot be migrated to the old logic for migrating NativeScript projects is not needed any more.
Also the server returns NativeScript.json with additional property - obsoleteVersions and the property npmVersions is renamed to versions.

The improvement is described here http://teampulse.telerik.com/view#item/312022

### CODE REVIEW

**Comment by TsvetanMilanov** [comment]
run ci

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "appbuilder",
   "preferGlobal": true,
-  "version": "3.2.0",
```
Comment (rosen-vladimirov) [inline package.json]: why is this changed?

**Comment by rosen-vladimirov** [comment]
we should delete the code related to manual migration of `styles.xml`. Can you please delete it as well?

**Comment by rosen-vladimirov** [comment]
Cannot be merged without  https://github.com/Icenium/Ice/pull/5590

**Comment by rosen-vladimirov** [comment]
run ci

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by TsvetanMilanov** [comment]
run ci
